client/src/components/closedTasksController.js

```diff
@@ -0,0 +1,16 @@
+function recentTasksController($http) {
```

Comment:
file is named "closedTasksController" while controller function is named "recentTasksController". Please use the equal naming to avoid confusion.